test/bats/cli-test.bats

```diff
@@ -35,6 +35,11 @@ load helpers
     assert_cmd_verify_failure
 }
 
+@test "schemavalidator verifier test" {   
+    run bin/ratify verify -c $RATIFY_DIR/config.json -s wabbitnetworks.azurecr.io/test/all-in-one-image:signed
```

Comment:
Apologies if this has been asked already. Just catching up a bit. Would the all in one test below on L43 take care of this since it's the same image? Is it just to add a separate test case that will show up in the run?

Reply:
You are correct it's a redundant test. I used the all-in-one for now as there is not a image with a unique artifact type (not assigned to another verifier) that can be validated against. I added a comment so we remember to update this once we have an image tagged for this verifier available. Does that work or should I add an issue to put the test back once a tagged image is available?

Reply:
Thanks for the clarification. Makes sense. I think it should be fine to keep the extra test so there's separation.